install-dev/upgrade/sql/1.7.3.0.sql

```diff
@@ -3,3 +3,16 @@ SET NAMES 'utf8';
 
 UPDATE `PREFIX_tab` SET `position` = 0 WHERE `class_name` = 'AdminZones' AND `position` = '1';
 UPDATE `PREFIX_tab` SET `position` = 1 WHERE `class_name` = 'AdminCountries' AND `position` = '0';
+
+CREATE TABLE IF NOT EXISTS `PREFIX_store_lang` (
+  `id_store` int(11) unsigned NOT NULL,
+  `id_lang` int(11) unsigned NOT NULL,
+  `name` varchar(128) NOT NULL,
```

Comment:
Please change varchar(128) to varchar(255) everywhere. I can see no reason to limit it to 128 characters.